### PR TITLE
add mentoring info page

### PR DESCRIPTION
**Pull request by filtercake**
also add link on host page. see #286.

PTAL @martinheidegger @finnp @SomeoneWeird

### CODE REVIEW

File: mentor-at-event.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+
+
+    <div class="container" style="background-color: #fff;">
+      <div class="two-thirds centered">
+          <p data-i18n="mentoring-intro-part1">
+            So an organizer has asked you to mentor at a NodeSchool event, but you're a bit unsure what you're getting into or if can do it? No worries :)
+          </p>
+          <p data-i18n="mentoring-intro-part2">
+            First of all, if you have been asked about mentoring, you probably have some time of experience in JS and node. Students will likely ask questions you have had yourself beack then.
```
Comment (martinheidegger) [inline mentor-at-event.html]: Beack

**Comment by finnp** [comment]
I think the message on this page is great. I like the text and it also looks good with that image and the quote.

However putting this on the website means, we have to translate it into all the different languages. I am wondering if it is worth to put it up there or whether it would be enough to just create a document in the [organizers repository](https://github.com/nodeschool/organizers). Right now it's also quite hidden (3rd level), so not many people will probably actually read it.

Anyone else wants to give input on this?

**Comment by filtercake** [comment]
> whether it would be enough to just create a document in the organizers repository

makes total sense, will do that!

thx for the suggestion :)

**Comment by martinheidegger** [comment]
Having not-translated content on the page is no problem imho. English information is better than non information. If someone feels like translating they should do it :) 

Good page, good effort imho.

**Comment by martinheidegger** [comment]
@filtercake Rebase would be nice :)

**Comment by SomeoneWeird** [comment]
ack

**Comment by filtercake** [comment]
sorry, sth went wrong when rebasing and the PR got automatically closed. looks like I need to open a new one...

**Pull request closed without merging**